python/whylogs/core/metrics/condition_count_metric.py

```diff
@@ -67,13 +67,19 @@ class Condition:
 @dataclass(frozen=True)
 class ConditionCountConfig(MetricConfig):
     conditions: Dict[str, Condition] = field(default_factory=dict)
+    exclude_from_profile: bool = False
```

Comment:
Suggest: rename to something like "exclude_from_serialization"

consider moving to base class metric config.